modules/stanford_profile_styles/lib/scss/base/views/search.scss

```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
+@import '../../config';
+
+.page-content {
+  h1 {
+    margin-top: 9.5rem;
```

Comment:
Our convention is to apply margin on the bottom of elements. Can this statement be moved to a different element with a margin-bottom instead?

Reply:
I just used the same approach from stanford_basic https://github.com/SU-SWS/stanford_basic/blob/8e445c84fda48bf969bb446c2843039f9fb79be2/src/scss/components/stanford-page/_page-title.scss#L8

Reply:
Given the conversation in dev chat today, are we letting this go through?

Reply:
yeah i believe so.